src/api/todos.ts

```diff
@@ -0,0 +1,14 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
```

Comment:
Consider adding a comment above the function 'getTodos' to describe its purpose and functionality.